R/utility.R

```diff
@@ -65,7 +65,7 @@ readJson <- function(x, simplifyVector = TRUE, ...) {
 }
 
 writeJson <- function(x, file, auto_unbox = TRUE, pretty = TRUE, ...) {
```

Comment:
If you wanted to apply `na = "null"` to all JSON output files, you could treat it the same as other default arguments like `auto_unbox` and `pretty`. Add `na = "null"` to `writeJson()` and `na = na` to `jsonlite::write_json()`

Reply:
yes